index.html

```diff
@@ -1,15 +1,47 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="stylesheet" href="style.css">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins:wght@300&family=Roboto&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css" />
     <title>Robert Kanyoro</title>
-</head>
-<body>
-    <header class="top_Section">
-
-    </header>
-</body>
-</html>
\ No newline at end of file
+  </head>
+  <body>
+    <main class="top_Section">
+      <section class="header">
+        <div class="app_bar">
+          <p>9:41</p>
+          <ul class="icons_phone">
+            <li><img src="icons/Cellular Connection.png" alt="Network Cell"></li>
+            <li><img src="icons/Wifi.png" alt="wifi"></li>
+            <li><img src="icons/Battery.png" alt="Battery"></li>
+          </ul>
+        </div>        
+        <nav class="menu_icon">
+          <ul>
+            <li>My Logo</li>
+            <li><img src="icons/Union.png" height="12px" width="18px" alt="Menu"></li>
```

Comment:
- [ ] Please I suggest wraaping the hamburger menu with `nav` tag for better HTML semantics to improve user experience with mobile devices. Please [check this link](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page) for more information. 😄